### PR TITLE
Unstable mutagen now enlarges cockroaches

### PR DESCRIPTION
**Pull request by unid15**
Big roaches are slowish, have 35 health and 5-10 attack. They're similar to other hostile mobs - they don't lay eggs or do other cockroach stuff. Instead they kick your ass

When sprayed, the roach has a 10% chance to evolve. If that fails, there's a 20% chance to die. If that fails, there's a 0.5% chance to ascend.

Mutagen also very rarely creates cockroach matriarchs. They are bigger and stronger, and they'll attack unconscious mobs to create large roach eggs.

hmm

fuck

Also nerfs cockroach meat (5 nutriment -> 0.5, 5-12 chitin -> 2-6), however big cockroaches drop 11 nutriment and ~40 chitin 

:cl:
 * rscadd: Cockroaches will no longer tolerate getting sprayed with unstable mutagen, and WILL fight back.
 * rscadd: Added cockroach matriarchs. A rare sight to behold, they're proficient at laying eggs - if fed properly.
 * tweak: Nerfed the amount of nutriment and chitin drops from small cockroaches. Larger ones, however, drop much more.

### CODE REVIEW

**Comment by ghost** [comment]
absolutely disgusting

**Comment by Irradiation** [comment]
Jesus christ how terrifying.

**Comment by iFlashYou** [comment]
Is this the next non-antag griff tool after bees?

**Comment by ShiftyRail** [comment]
Please for the love of God no

**Comment by SonixApache** [comment]
patrolling the station almost makes you wish for a supermatter cascade

**Comment by Chazach** [comment]
absolutely based

**Comment by Aurx** [comment]
Post logs.

**Comment by spoonylove** [comment]
they should still lay eggs which inherit the growth for a fun pastime

**Comment by Chazach** [comment]
this just makes me want a xenobotany tier method for fucking with animal genes

**Comment by ghost** [comment]
Enlarge Your RADICAL STEVE

**Comment by unid15** [comment]
Here are cockroach matriarchs

fuck

They create big eggs by attacking unconscious people
This sprite is even worse but I may maybe create a better one tomorrow (or at least animate this one)

**Comment by ShiftyRail** [comment]
How many Pomfcoins do I need to spend to make this PR disappear

**Comment by spoonylove** [comment]
Make them affected by phazon as well if they arent already like bees.

**Comment by Jubisloviu** [comment]
the PR that saved /vgstation/

**Comment by spoonylove** [comment]
cucacaracha

put creatine on them for this to happen.

**Comment by iFlashYou** [comment]
Better idea, add them to the random mob event

**Comment by IzayaChan** [comment]
I want beekeepers to live in fear of the greyshirt

File: code/modules/mob/living/simple_animal/hostile/bigroach.dm [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	speak_chance = 1
+
+	move_to_delay = 4
+
+	maxHealth = 35
+	health = 35
+
+	size = SIZE_SMALL
+
+	minbodytemp = 273.15		//Can't survive at below 0 C
```
Comment (ShiftyRail) [inline code/modules/mob/living/simple_animal/hostile/bigroach.dm:22]: Time to regulate the temps at -1°C every time they are spotted

**Comment by Sooose** [comment]
This is absolutely awful, when is it getting merged? Also, do they have a limit on how many matriachs we can have like we do spider queens?

**Comment by unid15** [comment]
Now there's a global limit on the amount of big roaches (50 at any given time). Matriarchs have no limit, but they will never spawn if there's already a matriarch somewhere nearby (7 tile range), and they're pretty rare (the chance of one spawning is pretty small)

**Comment by Blithering** [comment]
Do they inherit names when they evolve? radical steve has to stay radical

**Comment by unid15** [comment]
They sure do!

**Comment by Shadowmech88** [comment]
Should this mutation be triggered by high doses of radiation as well?

**Comment by Sooose** [comment]
Yes and change it so radical Steve's room is unshielded.

**Comment by Killette2** [comment]
So what are bees getting from this? Cool bees when

**Comment by ShiftyRail** [comment]
@Sooose this just gives me more icnentive to rush Steve's room and take care of that problem

**Comment by SonixApache** [comment]
jesus christ how horrifying

**Pull request merged**